submissions/sofiichuk/pop-up-task/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    list-style: none;
+}
+
+.navBar>ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+    justify-content: end;
+    margin: 20px 40px 0 0;
+ }
+
+a, span {
+    text-decoration: none;
+    color: #4f4f4f;
+    width: 60px;
+    font-family: sans-serif;
+}
+
+.dropdownMenu a, span {
+    display: block;
+}
+
+img {
+    height: 35px;
+    width: auto;
+}
+
+img[src="icons/bell.png"] {
+    height: 28px !important;
+    width: 28px !important;
+}
+
+input {
+    position: absolute;
+    appearance: none;
+    -webkit-appearance: none;
+    cursor: pointer;
+    height: 37px;
+    width: 37px;
+}
+
+label {
+    display: block;
+}
+
+.dropdownMenu>ul {
+    max-height: 0;
+    display: none;
+    position: absolute;
+}
+
+#input_1:checked ~ .dropdownMenu>ul {
+    max-height: 500px;
+    top: 60px;
+    right: 52px;
+    gap: 30px;
+    margin: 15px;
+    position: absolute;
+    display: grid;
+    grid-template-columns: repeat(3, 60px);
+    grid-template-rows: repeat(4, 40px) 15px repeat(2, 40px);
+    font-size: 12px;
+    text-align: center;
+}
+
+.showMoreButton {
+    width: 240px;
+    display: block;
+}
+
+.dropdownMenu>ul>li:nth-child(13) {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid using too specific CSS selectors. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
Replaced all selectors with either classes or id’s. But left tags in cases, when targeting each and every element: e.g. display: block for all anchors… the same size for all images…